cmd/root.go

```diff
@@ -39,4 +41,7 @@ func Execute() {
 
 func init() {
 	rootCmd.SetHelpCommand(&cobra.Command{Use: "no-help", Hidden: true})
+	rootCmd.PersistentFlags().BoolP("quiet", "q", false,
+		"Only shows SUCCESS/FAILURE as result of command execution instead of showing all output of command")
```

Comment:
Description could be better - 

What do you think about `Hide output log of command & only show SUCCESS/FAILURe result`?

Reply:
will change description.